src/runtime_src/core/edge/drm/zocl/zocl_xclbin.c

```diff
@@ -902,6 +902,8 @@ zocl_xclbin_load_pskernel(struct drm_zocl_dev *zdev, void *data)
         int ret = 0;
         int count = 0;
 	void *aie_res = 0;
+	struct device_node *aienode;
```

Comment:
Uninitialized variable.
```suggestion
	struct device_node *aienode = NULL;
```